TESTING.md

```diff
@@ -18,21 +18,28 @@ They live in the `tests` directories of each app. Run them with grunt: `grunt un
 ## Stack overview
 
 ### Requirements
-Follow the guide [DEVELOPMENT.md](DEVELOPMENT.md)
-JDK installed for Selenium.
-Docker to run couchdb.
+
+1. Follow the guide [DEVELOPMENT.md](DEVELOPMENT.md)
+2. JDK installed for Selenium.
+3. Docker to run couchdb.
 
 ### Local Run
 
 `grunt e2e` installs and runs chromedriver, starts couchdb in docker, pushes the compiled app to couchdb, starts api, starts sentinel, and then runs protractor tests against your local environment. 
 
-### WebdriverIO Local Run
+### WebdriverIO
+
+#### Run locally
+
+1. Run `npm ci`
+2. Run `grunt`
+3. Run `npm run wdio-local`
+
```

Comment:
Not a major, but there is a step here about Viewing the report on local run with `npx allure open`

Reply:
Good catch - fixed.